.github/workflows/continuous-delivery.yml

```diff
@@ -43,7 +49,7 @@ jobs:
           INPUT_MERGE: ${{ github.event.inputs.merge }}
         run: |
           if [ "$GITHUB_REF_NAME" = 'master' ]; then
-            if [ "${INPUT_MERGE,,}" = 'y' ]; then
+            if [ "$INPUT_MERGE" = true ]; then
```

Comment:
yes, surprisingly, this is how to evaluate boolean variables in shell... https://stackoverflow.com/a/2953673

Reply:
We were looking at the same site :)